### PR TITLE
Add stuff

### PR DESCRIPTION
**Pull request by UnknownSilicon**

### CODE REVIEW

File: src/org/robockets/steamworks/RobotMap.java [hunk 1]
```diff
@@ -19,6 +17,8 @@
     public static TalonSRX conveyorSpeedControllerOne = new TalonSRX(PortNumbers.SHOOTER_CONVEYOR_SC_ONE);
     public static TalonSRX conveyorSpeedControllerTwo = new TalonSRX(PortNumbers.SHOOTER_CONVEYOR_SC_TWO);
 
+    public static ADXRS450_Gyro gyro = new ADXRS450_Gyro(SPI.Port.kOnboardCS0);
```
Comment (simon-andrews) [inline src/org/robockets/steamworks/RobotMap.java]: Consider moving `SPI.Port.kOnboardCS0` to its own variable.

Comment (UnknownSilicon) [reply src/org/robockets/steamworks/RobotMap.java]: Why? It is already a variable in SPI.Port

File: src/org/robockets/steamworks/RobotMap.java [hunk 2]
```diff
@@ -1,8 +1,6 @@
 package org.robockets.steamworks;
 
-import edu.wpi.first.wpilibj.Encoder;
-import edu.wpi.first.wpilibj.RobotDrive;
-import edu.wpi.first.wpilibj.TalonSRX;
+import edu.wpi.first.wpilibj.*;
```
Comment (simon-andrews) [inline src/org/robockets/steamworks/RobotMap.java]: I bet Eclipse wouldn't have this problem. :smirk:

Comment (UnknownSilicon) [reply src/org/robockets/steamworks/RobotMap.java]: I don't know where that came from, I blame Intellij

**Review by simon-andrews — CHANGES_REQUESTED** [review]
Overall, looks like great stuff but needs a few really minor changes.

File: src/org/robockets/steamworks/RobotMap.java (hunk 1 above)
Comment (simon-andrews) [reply src/org/robockets/steamworks/RobotMap.java]: To keep everything in one place. That's the whole point of `PortNumbers` and `RobotMap`, isn't it?

`PortNumbers` would have an `SPI.Port` named `gyroPort`.

**Comment by UnknownSilicon** [comment]
@simon-andrews Fixed

**Review by simon-andrews — APPROVED** [review]

**Pull request merged**